applications/tari_console_wallet/src/ui/components/notification_tab.rs

```diff
@@ -0,0 +1,73 @@
+use crate::ui::{components::Component, state::AppState};
+use tari_comms::runtime::Handle;
+// use tari_p2p::auto_update::SoftwareUpdaterService;
+use tui::{
+    backend::Backend,
+    layout::{Constraint, Layout, Rect},
+    style::{Color, Modifier, Style},
+    text::{Span, Spans},
+    widgets::{Block, Borders, Paragraph, Wrap},
+    Frame,
+};
+
+pub struct NotificationTab {}
+
+impl NotificationTab {
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    fn draw_notifications<B>(&mut self, f: &mut Frame<B>, area: Rect, app_state: &AppState)
+    where B: Backend {
+        let block = Block::default().borders(Borders::ALL).title(Span::styled(
```

Comment:
I've added some explanation at the top of the file. So that everyone can easily see the proper file where the add_notification is called. Let me know if you think it's ok, or should change it.
I also added some TODOs (from top of my head) that we can implement in the future. If you have any ideas for other TODOs let me know, or you can create PR after this one.

Reply:
Sorry, I meant in the UI above the notifications block, info for the user - but it's not critical

Reply:
Oh, I see. To me the "notifications" is self-explanatory, but as a programmer I'm biased. But we should definitely be very careful with the message that we added. We can add the explanatory on the top later when will have more types of notification, not just update :-)